app/models/voxelytics/LokiClient.scala

```diff
@@ -28,7 +28,8 @@ class LokiClient @Inject()(wkConf: WkConf, rpc: RPC, val system: ActorSystem)(im
 
   private val POLLING_INTERVAL = 1 second
   private val LOG_TIME_BATCH_INTERVAL = 1 days
-  private val LOG_ENTRY_BATCH_SIZE = 5000L
+  private val LOG_ENTRY_QUERY_BATCH_SIZE = 5000L
```

Comment:
```suggestion
  private val LOG_ENTRY_QUERY_BATCH_SIZE = 5000
```
(I guess there is no real reason to have this be a Long? I’d change it for consistency with the new constant)